data/api/client-server/profile.yaml

```
@@ -244,44 +273,46 @@ paths:
                   avatar_url:
                     type: string
                     format: uri
```

To improve the rendering slighty:

```suggestion
                    format: mx-mxc-uri
                    pattern: "^mxc:\\/\\/"
```

---

Thanks! I've updated that here: https://github.com/matrix-org/matrix-spec/pull/2071/commits/30fd627a00a941b69ec67efcbd72ae016c0eae77